rasa/nlu/persistor.py

```diff
@@ -28,6 +28,20 @@ def get_persistor(name: Text) -> Optional["Persistor"]:
             os.environ.get("AZURE_ACCOUNT_KEY"),
         )
 
+    if name == "ali-oss":
+        from ali_persistor import OssPersistor
+
+        return OssPersistor(
+            access_key_id=os.environ.get("ALI_ACCESS_KEY_ID"),
+            access_key_secret=os.environ.get("ALI_ACCESS_KEY_SECRET"),
+            sts_role_arn=os.environ.get("ALI_STS_ROLE_ARN"),
+            region_id=os.environ.get("ALI_REGION_ID"),
+            endpoint=os.environ.get("ALI_ENDPOINT"),
+            bucket_name=os.environ.get("ALI_BUCKET_NAME"),
+            bucket_path=os.environ.get("ALI_BUCKET_PATH"),
+            local_path=os.environ.get("ALI_LOCAL_PATH"),
+        )
+
```

Comment:
@HawkeWoo I thought of a more general solution like the one below. It has the advantage that this also works for other users who provide their own persistors.
```suggestion
        try:
            persistor = rasa.utils.common.class_from_module_path(name)
            return persistor()
        except ImportError:
            raise ImportError(f"Unknown model persistor {name}. Please make sure to "
                              f"either use an included model persistor (`aws`, `gcs` "
                              f"or `azure`) or specify the module path to an external "
                              f"model persistor.")
```

Reply:
> @HawkeWoo I thought of a more general solution like the one below. It has the advantage that this also works for other users who provide their own persistors.

That sound good, I had change my code and import the persistor by module path.
For example, we can use aliyun oss persistor like:
`rasa run --remote-storage ali_persistor.OssPersistor`
and the external persistor class will use the environment variables: `RASA_PERSISTOR` to init  if nesserary.